tests/system_tests/test_core/test_start_method.py

```diff
@@ -3,72 +3,72 @@
 import platform
 
 import pytest
+import wandb
 from wandb.errors import UsageError
 
 
-def test_default(relay_server, wandb_init):
-    with relay_server() as relay:
-        run = wandb_init()
-        run_id = run.id
-        run.log(dict(val=1))
-        run.finish()
+def test_default(wandb_backend_spy):
+    run = wandb.init()
+    run_id = run.id
+    run.log(dict(val=1))
+    run.finish()
 
-    summary = relay.context.get_run_summary(run_id)
-    telemetry = relay.context.get_run_telemetry(run_id)
-    assert summary["val"] == 1
-    assert telemetry and 5 in telemetry.get("8", [])
+    with wandb_backend_spy.freeze() as snapshot:
+        summary = snapshot.summary(run_id=run_id)
+        telemetry = snapshot.telemetry(run_id=run_id)
+        assert summary["val"] == 1
+        assert telemetry and 5 in telemetry.get("8", [])
 
 
-def test_junk(relay_server, wandb_init):
-    with relay_server():
-        with pytest.raises(UsageError):
-            run = wandb_init(settings=dict(start_method="junk"))
-            run.finish()
+def test_junk():
+    with pytest.raises(UsageError):
+        run = wandb.init(settings=dict(start_method="junk"))
```

Comment:
how does it work? without a user?

Reply:
I believe this is working without a user because the `start_method` is being validate before any actual work in wandb.init is done.

[Here](https://github.com/wandb/wandb/blob/main/wandb/sdk/wandb_init.py#L1230) is where the settings dict gets converted to a settings object and then inside the settings object there is some validation being done.